web/src/i18n/locales/en.json

```diff
@@ -48,6 +48,13 @@
         "Username": "Username",
         "You must open the link from the same device and browser that initiated the registration process": "You must open the link from the same device and browser that initiated the registration process",
         "You're being signed out and redirected": "You're being signed out and redirected",
-        "Your supplied password does not meet the password policy requirements": "Your supplied password does not meet the password policy requirements."
+        "Your supplied password does not meet the password policy requirements": "Your supplied password does not meet the password policy requirements.",
+        "Use OpenID to verify your identity": "Use OpenID to verify your identity",
+        "Access your display name": "Access your display name",
+        "Access your group membership": "Access your group membership",
+        "Access your email addresses": "Access your email addresses",
+        "Accept": "Accept",
+        "Deny": "Deny",
+        "The above application is requesting the following permissions": "The above application is requesting the following permissions"
```

Comment:
@mind-ar feel like translating those before I merge this?

Reply:
Keep in mind `OpenID` is a terminology that shouldn't be translated.

Reply:
Hi @james-d-elliott , just created https://github.com/authelia/authelia/pull/2825 pointing to this branch 
I don't know if this is the correct way to do a contribution over a branch other than master
let me know if i need to use another method

Reply:
Yep that should be fine.